internal/cache/file/cache_handle.go

```diff
@@ -195,14 +196,13 @@ func (fch *CacheHandle) IsSequential(currentOffset int64) bool {
 }
 
 // Close closes the underlined fileHandle points to locally downloaded data.
-func (fch *CacheHandle) Close() (err error) {
```

Comment:
why are we changing this behavior ?

Reply:
As discussed, we will keep the behavior like random_read. That means, Close() will return an error but we will not escalate that error instead debug as warning.